### DISCUSSION TITLE
Document urls not loading for some users

### DISCUSSION THREAD

**Discussion by SWDmarketingSquad** [community/community#36585] [Other Feature Feedback, Questions, & Ideas]
Hi,

We are having trouble with our documents not loading on our app. The links to our documents are embedded in the html code by our documents pdf urls on our website and are updated automatically whenever we update the pdfs through our website. For some reason a few of our users cannot access these documents on the app, the documents will not load. Which is weird because for other users the documents will load and have no problem accessing the pdfs. Has anyone had a similar issue or have any insight on what could be causing this?

**Comment by ossanna16** [comment]
Hi @SWDmarketingSquad ,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on web development. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
